.github/workflows/main.yml

```diff
@@ -245,7 +253,7 @@ jobs:
           $describe = (git describe --always --tags --long --match 'v*')
           $ver_arr = $describe -split '[-v]'
           # Always ".0" to make it windows store compatible
-          $ver_str = $ver_arr[1] + ".0"
+          $ver_str = $ver_arr[1] + ".2"
```

Comment:
Windows doesn't let you reinstall the same package version, and if I uninstall the old version, the gigabyte plus of content is uninstalled and must be redownloaded.

This patch is why the PR is marked as a Draft.